chia/util/chia_version.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from packaging.version import InvalidVersion, Version
+
+from chia import __version__
+
+
+def chia_version_str() -> str:
+    return chia_version_str_from_version(Version(__version__))
+
+
+def chia_version_str_from_version(version: Version) -> str:
```

Comment:
One could overload this function - I might also suggest chia_short_version() - I think something like this would work and is correct

@overload
def chia_short_version(version: Version) -> str:

@overload
def chia_short_version(version: str) -> str:

Reply:
As python doesn't have real overload support and we need a function which handles all cases anyway i just opted for an optional union type. Let me know if you prefer this over the previous approach